app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart

```diff
@@ -134,15 +142,15 @@ class FileFlakyIssueAndPR extends ApiRequestHandler<Body> {
     return target != null && target[kCiYamlTargetIsFlakyKey] == true;
   }
 
-  bool _getIgnoreFlakiness(String builderName, YamlMap ci) {
-    final YamlList targets = ci[kCiYamlTargetsKey] as YamlList;
-    final YamlMap? target = targets.firstWhere(
-      (dynamic element) => element[kCiYamlTargetNameKey] == builderName,
-      orElse: () => null,
-    ) as YamlMap?;
-    bool ignoreFlakiness = target?[kCiYamlPropertiesKey][kCiYamlTargetIgnoreFlakiness] != null &&
-        target?[kCiYamlPropertiesKey][kCiYamlTargetIgnoreFlakiness] == 'true';
-    return target != null && ignoreFlakiness;
+  bool _getIgnoreFlakiness(String builderName, CiYaml ciYaml) {
+    final Target target;
+    try {
```

Comment:
There is a test('Can file issue but not pr for test not found in ci.yaml') that fails here (and needs to be wrapped in the try...on), since there is no target returned by singleWhere. Is this a case we we would in reality? If so, should wrap this call in the other file as well.

Reply:
@keyonghan do you know why we need this check? This seems to be before flutter/infra rolled ci.yaml, and is no longer necessary

Reply:
done.